app/components/PickBakerStakeAmount.tsx

```diff
@@ -24,6 +36,41 @@ interface Props {
     hasPendingChange?: boolean;
 }
 
+/**
+ * Check whether the proposed stake will cause the account to breach the capital bound.
+ */
+function useCapitalBoundCheck(
+    accountInfo: AccountInfo | undefined,
+    stake: string
+): { showWarning: true; limitAfterUpdate: bigint } | { showWarning: false } {
+    const rewardStatus = useAsyncMemo(getRewardStatusLatest);
+    const capitalBound = useCapitalBound();
+
+    if (
+        !capitalBound ||
+        !isValidCcdString(stake) ||
+        !rewardStatus ||
+        !isRewardStatusV1(rewardStatus)
+    ) {
+        return { showWarning: false };
+    }
+    const newStake = ccdToMicroCcd(stake);
+    const currentStake =
+        accountInfo && isBakerAccount(accountInfo)
+            ? accountInfo.accountBaker.stakedAmount
+            : 0n;
+    const newTotalStake =
+        rewardStatus.totalStakedCapital - currentStake + newStake;
+    const limitAfterUpdate =
```

Comment:
This limit calculation has to also consider any delegated amounts.

Reply:
No, the limit has nothing to do it. I have fixed it, so the delegated amount is taken into consideration when checking if the bound has been violated.